guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -34,7 +35,8 @@ Default is `Default_Organization_View`.
 . From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
 . Click *Update*.
 . In the {ProjectWebUI}, navigate to *Content* > *Products*.
-. Click *Sync Now* to synchronize the repositories.
+. Select the Product that contains the repositories that you want to synchronize.
+. From the *Select Action* menu, select *Sync Now* to synchronize the repositories.
```

Comment:
```suggestion
. Click *Sync Now* to synchronize the repositories.
```
There actually is a **Sync Now** button that the user can use.
![image](https://github.com/theforeman/foreman-documentation/assets/20555018/59a04630-2f12-44f3-b30b-2d4a2929e53d)
This is Sat 6.15.